src/uct/ib/mlx5/gdaki/gdaki.cuh

```diff
@@ -347,11 +347,12 @@ template<ucs_device_level_t level>
 UCS_F_DEVICE ucs_status_t uct_rc_mlx5_gda_ep_put_single(
         uct_device_ep_h tl_ep, const uct_device_mem_element_t *tl_mem_elem,
         const void *address, uint64_t remote_address, size_t length,
-        unsigned cid, uint64_t flags, uct_device_completion_t *comp)
+        unsigned channel_id, uint64_t flags, uct_device_completion_t *comp)
 {
     auto ep       = reinterpret_cast<uct_rc_gdaki_dev_ep_t*>(tl_ep);
     auto mem_elem = reinterpret_cast<const uct_rc_gdaki_device_mem_element_t*>(
             tl_mem_elem);
+    auto cid      = channel_id % ep->num_channels;
```

Comment:
can we ensure (round up/down) ep->num_channels to power of 2, and use bitwise instead of %?